webapp/utilities/droitsRateLimiter.js

```diff
@@ -0,0 +1,12 @@
+const { RateLimiterMemory } = require("rate-limiter-flexible");
+
+const MAX_REQUESTS = 10;
+const WINDOW_MS = 60 * 1000;
```

Comment:
I think we can have `DURATION_IN_SECONDS = 1` instead of `WINDOW_MS = ...`

I've been scouring the documentation and it seems to be in seconds